cmd/restic/cmd_dump.go

```diff
@@ -63,6 +64,7 @@ func init() {
 	initSingleSnapshotFilter(flags, &dumpOptions.SnapshotFilter)
 	flags.StringVarP(&dumpOptions.Archive, "archive", "a", "tar", "set archive `format` as \"tar\" or \"zip\"")
 	flags.StringVarP(&dumpOptions.Target, "target", "t", "", "write the output to target `path`")
+	flags.BoolVar(&dumpOptions.CompressZipArchive, "compress-zip", false, "compress zip archive contents")
```

Comment:
Can we do a shorthand `c` as well? You can check if there any conflicts with the global options.